app/Services/EDocument/Standards/FatturaPANew.php

```diff
@@ -85,6 +114,10 @@ private function setDatiTrasmissione(): self
 
         $this->DatiTrasmissione->IdTrasmittente = $this->IdTrasmittente;
 
+        $contatti = new Contatti();
+        $contatti->Email = $this->invoice->company->settings->email;
+        $this->DatiTrasmissione->ContattiTrasmittente = $contatti;
```

Comment:
Property InvoiceNinja\EInvoice\Models\FatturaPA\DatiTrasmissioneType\DatiTrasmissione::$ContattiTrasmittente (InvoiceNinja\EInvoice\Models\FatturaPA\ContattiTrasmittenteType\ContattiTrasmittente) does not accept InvoiceNinja\EInvoice\Models\FatturaPA\Contatti. PHPStan[assign.propertyType](https://phpstan.org/error-identifiers/assign.propertyType)

I think there are type issues here